README.md

```diff
@@ -3,14 +3,24 @@
 [![npm](https://img.shields.io/npm/v/trustlines-clientlib.svg)](https://www.npmjs.com/package/trustlines-clientlib)
 [![CircleCI branch](https://img.shields.io/circleci/project/github/trustlines-protocol/clientlib/master.svg)](https://circleci.com/gh/trustlines-protocol/clientlib)
 [![Codecov branch](https://img.shields.io/codecov/c/github/trustlines-protocol/clientlib/master.svg)](https://codecov.io/gh/trustlines-protocol/clientlib)
+[![gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/trustlines/community)
 
-A TypeScript/JavaScript library for interacting with the [Trustlines Network Protocol](https://trustlines.network/).
+The clientlib is a component of the [Trustlines Protocol](https://trustlines.foundation/protocol.html).
+The Trustlines Protocol is a set of rules to allow the transfer of value on top of existing trust
+relationships stored on a trustless infrastructure, here a blockchain.
 
-## Installation
+In the technology stack, the clientlib is located on top of a [relay server](https://github.com/trustlines-protocol/relay),
+and a mobile application can be built on top of it.
 
-### Install via package manager
+The goal of the clientlib is to provide a user focused means to interract with Trustlines via a relay server.
+The intent is to make it as easy as possible to build a user interface for Trustlines on top of it.
 
-We publish the trustlines-clientlib to npm as a ES6 module.
+## Get Up and Running
+
+[// todo write pre-requisites, is it obvious already when we say es6 module?]: #
```

Comment:
Yes, I think it we can omit that. It should be quite obvious and other JS libs I looked at also do not really specify the pre-requisites.